src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts

```diff
@@ -176,6 +184,18 @@ export class CreateScenariosComponent implements OnInit {
     });
   }
 
+  private constraintsFormValidator(constraintsForm: AbstractControl): boolean {
+    // Only one of budget or max treatment percentage is required.
+    const maxBudget = constraintsForm.get('budgetForm.maxBudget');
+    const optimizeBudget = constraintsForm.get('budgetForm.optimizeBudget');
+    const maxArea = constraintsForm.get('treatmentForm.maxArea');
+    console.log(constraintsForm.value);
+    console.log(
```

Comment:
are we intentionally keeping the logs for now?

Reply:
haha nope, thanks for catching!